README.md

```diff
@@ -1,22 +1,32 @@
 # Duqhan
```

Comment:
you should change the name of icon from "Duqhun" to "Duqhan", i think it is all over the place, you might want to rename it everywhere.